### DISCUSSION TITLE
[Question]: Missing previous recordings after Frigate restart

### DISCUSSION THREAD

**Discussion by ronniekw** [blakeblackshear/frigate#10088] [Ask A Question] [answered]
### What is your question:

After restarting the Frigate Docker, all previous recordings and events disapper from Frigate GUI. Only the recordings after the restart is shown. Is that by design or is it possible to show older recordings up to the 7 day limit I have set even after a restart.

**Comment by NickM-27** [comment]
this usually indicates an error in the docker config

**Reply by ronniekw** [reply]
I can't spot any errors in docker-compose.xml

version: "3.9"
services:
  frigate:
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    image: ghcr.io/blakeblackshear/frigate:stable
    shm_size: "64mb"
    devices:
      - /dev/bus/usb:/dev/bus/usb #Used for Coral if Available
      - /dev/video10 #Map the Pi hardware acceleration
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /home/ronnie/frigate-nvr/config.yml:/config/config.yml
      - /media/ronnie/Elements1/frigate-recordings:/media/frigate     # my external USB disk
      - type: tmpfs #Remove this if using a Pi with 2GB or Less RAM
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "8554:8554" # RTSP feeds
      - "8555:8555/tcp" # WebRTC over tcp
      - "8555:8555/udp" # WebRTC over udp
    environment:
      FRIGATE_RTSP_PASSWORD: "********"

**Reply by NickM-27** [reply, marked as answer]
> - /home/ronnie/frigate-nvr/config.yml:/config/config.yml

you are only mapping your config file meanwhile the release notes and updated installation docs state that the full folder must be mapped `- /home/ronnie/frigate-nvr/:/config/`

**Reply by ronniekw** [reply]
Thanks :) Now it works!
